<ui version="4.0" >
 <class>ClipboardWidget</class>
 <widget class="QWidget" name="ClipboardWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Clipboard</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="margin" >
    <number>2</number>
   </property>
   <item row="0" column="0" colspan="3" >
    <widget class="QListView" name="listView" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="iconSize" >
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="resizeMode" >
      <enum>QListView::Adjust</enum>
     </property>
     <property name="spacing" >
      <number>5</number>
     </property>
     <property name="uniformItemSizes" >
      <bool>true</bool>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QPushButton" name="cmdRemoveSelected" >
     <property name="text" >
      <string>Remove Selected</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="cmdClearAll" >
     <property name="text" >
      <string>Clear All</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>84</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
